changelogs/api-management-service.md

```diff
@@ -1,5 +1,39 @@
 # Azure API Management service changelog
 
+## Release - API Management service: October, 2022
+
+A regular Azure API Management service update was started on October 31, 2022. It may take several weeks for your API Management service to receive the update.
+
+### Feature retirements
+
+1. [Support for Azure API Management self-hosted gateway version 0 and version 1 ends 1 October 2023](https://azure.microsoft.com/updates/support-for-azure-api-management-selfhosted-gateway-version-0-and-version-1-ends-1-october-2023/).
+
+### New features, fixes, and improvements
+
+1. You can now easily validate Azure Active Directory tokens on incoming requests with the new `validate-azure-ad-token` policy. [Learn more](https://learn.microsoft.com/azure/api-management/api-management-access-restriction-policies#ValidateAAD).
```

Comment:
Our blog post is out on https://teams.microsoft.com/l/message/19:3d3ef9ae999146c5abb9d5f7d1a1f5bd@thread.skype/1668796971285?groupId=48dcdab4-70a0-46b2-a680-aa8e200a9126